examples/react-template/screens/Column.tsx

```
@@ -19,61 +19,121 @@ import {
+              </ColumnsItem>
+            )}
+          )}
+      </Columns>
 
+      <Divider/>
+      <Text> Columns is scrollable and fullbleed.</Text>
+      <Columns scrollable mobile fullBleed>
+        {
+          Object.values([1,2,3,4,5,6,7,8,9,10,11,12]).map( (colSize, index) => {
+            return (
+              <ColumnsItem size={colSize}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)